crates/cast/src/lib.rs

```diff
@@ -268,7 +268,6 @@ pub async fn wait_for_tx(
     tx_hash: FieldElement,
     retries: u8,
 ) -> Result<&str> {
-    println!("Transaction hash: {tx_hash:#x}");
```

Comment:
What was the reason to remove this?

Reply:
This would be outputted each time we do any kind of transaction, so after running the script there was a possibility that we would have multiple "transaction hash" messages, that say nothing. Now the user will have a control over it in deployment scripts, and for regular cast commands we output this info nonetheless